backend/api_calls.py

```diff
@@ -1055,6 +1056,29 @@ def MarkNotificationSeen(request, firebase):
       request['playerId'], request['notificationId']), put_data)
 
 
+def RegisterUserDevice(request, firebase):
+  """Register a user device to a userId.
+
+  Validation:
+    userId must exist.
+
+  Args:
+    userId: User id to associate with.
+    deviceToken: Ionic device token.
```

Comment:
Nah. There just wasn't anything that made sense to update.
Given that you *are* writing a method which updates the user data, is there any reason not to include the device in the original register vs requiring a register followed by a second RPC to update that data with the device?